File: src/domain/auth/exception/invalid-refresh-token.exception.ts
import { HttpStatus } from '@nestjs/common';

import { ApiException } from '@wink/swagger';

export class InvalidRefreshTokenException extends ApiException {
  constructor() {
    super({
      swagger: '잘못된 리프레시 토큰일 때',
      message: '올바르지 않은 리프레시 토큰입니다.',
      code: HttpStatus.UNAUTHORIZED,
    });
  }
}

File: src/main/java/com/github/kmu_wink/wink_official_backend/domain/auth/exception/InvalidRefreshTokenException.java
package com.github.kmu_wink.wink_official_backend.domain.auth.exception;

import com.github.kmu_wink.wink_official_backend.common.api.exception.ApiException;
import org.springframework.http.HttpStatus;

public class InvalidRefreshTokenException extends ApiException {

    public InvalidRefreshTokenException() {

        super(HttpStatus.UNAUTHORIZED, "유효하지 않은 리프레시 토큰입니다.");
    }
}
